base_local_planner/cfg/ObstacleSpeedLimiter.cfg

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+# DWA Planner configuration
+
+from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, double_t, int_t, bool_t
+gen = ParameterGenerator()
+
+gen.add("half_angle", double_t, 0, "The half angle in which to consider obstructions for speed limits", 1.5, 0, 3.2)
```

Comment:
this description is a little confusing to me

Reply:
Done